.ci/pipelines/chaos_test_zeebe.groovy

```diff
@@ -0,0 +1,139 @@
+final PROJECT = "zeebe"
+final CHAOS_TEST_NAMESPACE = "zeebe-chaos-test"
+final NOTIFY_EMAIL = "maxim.danilov@camunda.com" // TODO change me
```

Comment:
Do you still want to get these emails ? :smile:

Reply:
Nope, I'll change it in the end :) It's for testing now.